src/short-urls/short-urls.service.ts

```diff
@@ -35,6 +36,13 @@ export class ShortUrlsService {
 		return existingUrl;
 	}
 
+	async findAllUrls(user: User, includeExpired?: string): Promise<ShortUrl[]> {
```

Comment:
Add default value of `false`
```suggestion
	async findAllUrls(user: User, includeExpired: boolean = false): Promise<ShortUrl[]> {
```

Reply:
can we change includeExpired to boolean. In the class-validator, where we validate we can use class-transformer to change to boolean
```
import { Transform } from 'class-transformer';

export class FilterOptionRequest {
  @Transform(({ value }) => value === 'true')
  @IsBoolean()
  @IsOptional()
  readonly includeExpired: boolean;
}
```

Reply:
@arjandhakal, i tried your suggested way however it didn't work. Looks like `@Transform` doesn't seem to work is `@IsOptional()` decorator is used. Instead, I explicitly changed the type of expired parameter to boolean from string in controller. Please check my latest commit.